### ISSUE TITLE
Scenes with all channels set to zero don't light buttons

### ISSUE THREAD

**Issue by enbyted** [mcallegari/qlcplus#983]
Reproduce:

- Given any fixture is added
- Given Scene A and B exists
- Given Scenes A and B use a single channel of any fixture
- Given Scene A sets it to zero
- Given Scene B sets it to any non-zero value
- Given two Buttons A and B in solo-frame (it's not solo frame specific, but this illustrates issue nicely)
- Given Button A activates Scene A
- Given Button B activates Scene B
- I expect that pressing button B will activate (light up) Button B and deactivate Button A - **this works**
- I expect that pressing button A will activate (light up) Button A and deactivate Button B - **this doesn't**

It also happens if for example I have a chaser going through Scenes A and B and buttons attached to these function - button B will light up, A - won't.

I'll provide a test workspace as soon as I get my hands on working QLC+ (I noticed this last night on event that I've ran).

**EDIT:**

> - I expect that pressing button A will activate (light up) Button A and deactivate Button B - **this doesn't**

Button B deactivates as expected, it's button A that doesn't activate.

**Comment by janosvitok** [comment]
This might be related to the fact that setting HTP channel to zero is no-op. Nevertheless, it should at least deactivate button B.

**Comment by enbyted** [comment]
@janosvitok it deactivates button B. I'm sorry I didn't state it correctly.

> This might be related to the fact that setting HTP channel to zero is no-op. Nevertheless, it should at least deactivate button B.

Yes, I thought about that, but couldn't check it since I broke my Qt environment :c

This should be the demo console: https://pastebin.com/1ckrSX2V

BTW. I'm not sure if it's issue with my environment (which it might, cause I messed up Qt lib installs), but latest git version of QLC+ crashes when trying to start chaser from button. (for example when running chaser in provided console)
